ports/poppler/portfile.cmake

```
@@ -4,14 +4,14 @@ vcpkg_from_gitlab(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO poppler/poppler
     REF "poppler-${POPPLER_VERSION}"
-    SHA512 18649364dc407080941b7c4010c0f26c1ce825d9ec49ff8e9ef298c62afb8d5bb77cea6a5cd1a74615190f433c265613dba42a6b7fdd80c2b5f00d372a31d21d
+    SHA512 23fe0f5445896e3a0731a0f3eae6aac84dc8f4b0718e947ae3ee3492295720408738f0c706eb2e5cbaa41e854dafc70e5c59c9c70e7d78ab33c318b2e8d7e4ff
     HEAD_REF master
     PATCHES
         export-unofficial-poppler.patch
 )
 file(REMOVE "${SOURCE_PATH}/cmake/Modules/FindFontconfig.cmake")
 
-set(POPPLER_PC_REQUIRES "freetype2 libjpeg libopenjp2 libpng libtiff-4 poppler-vcpkg-iconv")
+set(POPPLER_PC_REQUIRES "freetype2 libjpeg libopenjp2 libpng libtiff-4 poppler-vcpkg-iconv lcms2")
```

```suggestion
set(POPPLER_PC_REQUIRES "freetype2 libjpeg libopenjp2 libpng libtiff-4 poppler-vcpkg-iconv")
```
because it is handled in line 50. (:+1:)